cupy/sparse/linalg/__init__.py

```diff
@@ -1,5 +1,18 @@
-# Functions from the following SciPy document
-# https://docs.scipy.org/doc/scipy/reference/sparse.linalg.html
+import sys
+import warnings
 
-# "NOQA" to suppress flake8 warning
-from cupyx.scipy.sparse.linalg.solve import lsqr  # NOQA
+import cupyx.scipy.sparse.linalg
+
+
+if (3, 7) <= sys.version_info:
+    def __getattr__(name):
+        if hasattr(cupyx.scipy.sparse.linalg, name):
+            warnings.warn(
```

Comment:
FYI: users will see this message (maybe only) when `from cupy.sparse.linalg import lsqr`.